### PR TITLE
Fix Jenkins blowup caused by hacking tests involving external programs

### PR DESCRIPTION
**Pull request by SHuang-Broad**
@droazen , would you please review, before I go out and get enough beer (I hope I won't have to buy another round with this fix)?

Thanks!

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunBWAMEMViaCommandLineTest.java [hunk 1]
```diff
@@ -28,24 +29,18 @@
...
-            s = getNonNullEnvironmentVariable("PATHTOBWA");
-            if(null==s) {
-                return null;
-            }else{
-                return Paths.get(s);
-            }
-        }else{
-            return Paths.get(s);
-        }
+        return s == null ? (null==(s=System.getenv("PATHTOBWA")) ? null : Paths.get(s)) : Paths.get(s);
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunBWAMEMViaCommandLineTest.java]: This is a little TOO compact! (hard to read). Can you move the `System.getenv("PATHTOBWA")` call back into a separate if statement before the `? :` line?

**Comment by droazen** [comment]
Back to @SHuang-Broad for a quick readability fix.

**Comment by SHuang-Broad** [comment]
As recommended, @droazen . Please take a look again. Thanks!

**Comment by droazen** [comment]
:+1: looks good, merging -- brown drink waived because environment variables are just not fair :)

**Pull request merged**